src/components/BottomBar/BottomBarCourseReview.vue

```diff
@@ -118,17 +118,17 @@ export default Vue.extend({
       const [subject, number] = this.courseObj.code.split(' ');
       return `https://www.cureviews.org/course/${subject}/${number}`;
     },
-    CUROverallRating() {
+    CUROverallRating(): string | number {
```

Comment:
Did vue 3 require adding these types or did you just notice this?

Reply:
Vue 3 seems to be more strict on these